xbmc/addons/interfaces/GUI/AddonCallbacksGUI.h

```diff
@@ -116,12 +116,15 @@ class CAddonCallbacksGUI
   static float        Control_SettingsSlider_GetFloatValue(void *addonData, GUIHANDLE handle);
   static void         Control_SettingsSlider_SetFloatInterval(void *addonData, GUIHANDLE handle, float fInterval);
 
+  //! @todo Remove iconImage and thumbnailImage arguments in future Kodi versions. Results in NOP now.
   static GUIHANDLE    ListItem_Create(void *addonData, const char *label, const char *label2, const char *iconImage, const char *thumbnailImage, const char *path);
   static const char * ListItem_GetLabel(void *addonData, GUIHANDLE handle);
   static void         ListItem_SetLabel(void *addonData, GUIHANDLE handle, const char *label);
   static const char * ListItem_GetLabel2(void *addonData, GUIHANDLE handle);
   static void         ListItem_SetLabel2(void *addonData, GUIHANDLE handle, const char *label);
+  //! @todo Remove SetIconImage callback in future Kodi versions. Results in NOP now.
   static void         ListItem_SetIconImage(void *addonData, GUIHANDLE handle, const char *image);
+  //! @todo Remove SetThumbnailImage callback in future Kodi versions. Results in NOP now.
   static void         ListItem_SetThumbnailImage(void *addonData, GUIHANDLE handle, const char *image);
```

Comment:
The `ListItem_SetIconImage` and `ListItem_SetThumbnailImage` if not called from header can be removed.